### PR TITLE
Search component added

### PR DESCRIPTION
**Pull request by BraneOpacic3ap**

### CODE REVIEW

File: src/app/items/containers/items.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<app-search></app-search>
```
Comment (aberonja) [inline src/app/items/containers/items.component.html]: /n

File: src/app/items/components/search/search.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss']
+})
+export class SearchComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit() {}
+
+  onSearch(e) {
```
Comment (aberonja) [inline src/app/items/components/search/search.component.ts:13]: Do you want to leave this?

File: src/app/shared/top-header/top-header.component.html [hunk 3]
```diff
@@ -38,4 +38,4 @@
     </div>
   </div>
   <ds-page-header page-title="Accreditations"></ds-page-header>
-</header>
+</header>
```
Comment (aberonja) [inline src/app/shared/top-header/top-header.component.html]: /n

**Review by aberonja — APPROVED** [review]

File: src/app/items/components/search/search.component.ts (hunk 2 above)
Comment (BraneOpacic3ap) [reply src/app/items/components/search/search.component.ts:13]: yes.

**Pull request merged**